classes/04class/exercises/c04-rds01/luizfds/db-creation.txt

```diff
@@ -0,0 +1,114 @@
+aws ec2 create-security-group --group-name DevOpsAcademyRDS --description "DevOpsAcademyRDS"
+{
+    "GroupId": "sg-0e9fbfe79714d1db5"
+}
+
+curl http://checkip.amazonaws.com/
+101.116.46.174
+
+aws ec2 authorize-security-group-ingress \
+    --group-id sg-0e9fbfe79714d1db5 \
+    --protocol tcp \
+    --port 22 \
+    --cidr 101.116.46.174/24
+
+aws ec2 authorize-security-group-ingress \
+    --group-id sg-0e9fbfe79714d1db5 \
+    --protocol tcp \
+    --port 0-65535 \
+    --source-group DevOpsAcademyRDS
+
+aws rds create-db-instance \
+    --db-name devopsacademy \
+    --db-instance-identifier devopsacademy \
+    --allocated-storage 20 \
+    --vpc-security-group-ids sg-0e9fbfe79714d1db5 \
+    --db-instance-class db.t2.micro  \
+    --engine mysql \
+    --master-username luiz \
+    --master-user-password luizfds2020 \
+    --no-publicly-accessible
+{
+    "DBInstance": {
+        "PubliclyAccessible": false, 
+        "MasterUsername": "luiz", 
+        "MonitoringInterval": 0, 
+        "LicenseModel": "general-public-license", 
+        "VpcSecurityGroups": [
+            {
+                "Status": "active", 
+                "VpcSecurityGroupId": "sg-0e9fbfe79714d1db5"
```

Comment:
Did you reuse the same security group for RDS and EC2?

If so, it is better to split them and set RDS SG to allow ingress from EC2 SG on port 3306 only. EC2 SG would allow only SSH/22 to your IP address.

Reply:
I split my SG in two (DevOpsAcademyEC2 and DevOpsAcademyRDS), first one allow only access SSH on port 22 and second one on port 3306 for my IP address.